File: PrimeiroFormulario/app/src/main/java/com/fiap/primeiroformulario/MainActivity.java
package com.fiap.primeiroformulario;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText edtNome;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        edtNome = findViewById(R.id.edtNome);
    }


    public void enviar(View view) {
        String nome = edtNome.getText().toString();

        if (nome.trim().isEmpty()) {
            Toast.makeText(this, "Informe corretamente o nome", Toast.LENGTH_SHORT).show();
        } else{
            Toast.makeText(this, "Olá " + nome, Toast.LENGTH_SHORT).show();
        }
    }
}

File: 04InterfaceGrafica03/app/src/main/java/br/com/fiap/interfacegrafica03/MainActivity.kt
package br.com.fiap.interfacegrafica03

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import java.lang.Exception

class MainActivity : AppCompatActivity() {

    lateinit var edtNumero01 : EditText
    lateinit var edtNumero02 : EditText
    lateinit var txtResultado : TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        edtNumero01 = findViewById(R.id.edtNumero01)
        edtNumero02 = findViewById(R.id.edtNumero02)
        txtResultado = findViewById(R.id.txtResultado)
    }

    /**
     * Validação dos dados necessários para este formulário
     *
     * @param numero01
     * @param numero02
     *
     * @return Boolean
     */
    fun validar(numero01:String, numero02:String) : Boolean {
        if (numero01.trim().isEmpty()) {
            Toast.makeText(this,
                "Informe corretamente o primeiro número", Toast.LENGTH_SHORT).show()
            return false;
        }

        if ( numero02.trim().isEmpty() ) {
            Toast.makeText(this,
                "Informe corretamente o segundo número", Toast.LENGTH_SHORT).show()
            return false;
        }

        return true;
    }

    /**
     * Soma os números informados
     *
     * @param view Representa o componente que disparou o evento
     */
    fun somar(view: View) {
        if ( ! validar(edtNumero01.text.toString(), edtNumero02.text.toString()) ) {
            return
        }

        try {
            val numero01 = edtNumero01.text.toString().toInt()
            val numero02 = edtNumero02.text.toString().toInt()

            val soma = numero01 + numero02

            txtResultado.text = "Soma atual: ${soma}"
        } catch ( e:Exception ) {
            Toast.makeText(this,
                "Não foi possível somar os dados informados!", Toast.LENGTH_SHORT).show()
        }
    }
}